### ISSUE TITLE
wcag fail

### ISSUE THREAD

**Issue by krisztin** [krisztin/krisztin.github.io#17]
I mean, seriously...

![wcagfail.png](https://images.zenhubusercontent.com/5a9eb7a74b5806bc2bc99d2f/44da8528-cb59-4323-9993-057065c03293)
